crates/sats/src/satn.rs

```diff
@@ -340,6 +343,53 @@ impl<'a, 'b> ser::Serializer for SatnFormatter<'a, 'b> {
         })?;
         write!(self, ")")
     }
+
+    unsafe fn serialize_bsatn(self, ty: &crate::AlgebraicType, bsatn: &[u8]) -> Result<Self::Ok, Self::Error> {
```

Comment:
Can default implementations of supertrait methods meaningfully have doc comments? If so, I'd like to see a comment on this method (and the other impls here) which describe how they're implemented, and when implementors should overwrite them. If not, I'd put that comment on `trait Satn` instead.

Reply:
I misread (I blame the GitHub UI) and thought these were part of the above `trait Satn`.